src/scrape/scrape_job_test.go

```diff
@@ -200,6 +233,40 @@ func TestPopulateK8s(t *testing.T) {
 	}
 }
 
+func TestRestartCountDeltaValues(t *testing.T) {
+	intgr, err := integration.New("test", "test", integration.InMemoryStore())
```

Comment:
If I am not wrong two subsequent calls to integration.New() cause a panic (😢 )
If not really needed I would try to avoid using the integration in tests

Reply:
As integrations are globally, it would be advisable to have unique names for them. Otherwise, other test using `"test", "test"` might cause race/concurrency issues. The Test integration helper does this for you :)

https://github.com/newrelic/nri-kubernetes/blob/main/internal/testutil/integration.go#L23

Reply:
Thanks! I didn't know. I will fix it [here](https://github.com/newrelic/nri-kubernetes/blob/bc62e845386de80d99d1d5b928fc25d5b72ecd98/src/scrape/scrape_job_test.go#L195) too.

Reply:
Fixed here: [bc62e84](https://github.com/newrelic/nri-kubernetes/pull/382/commits/bc62e845386de80d99d1d5b928fc25d5b72ecd98)